packages/combobox/combobox.scss

```
@@ -248,6 +248,15 @@ $_combobox-search-input-selection-color--inverted: color.scale(
         }
     }
 
+    &--menu-closed {
+        .jkl-combobox__search-input {
+            position: absolute;
+            background-color: transparent;
+            width: 94%;
+            padding-right: 0;
+        }
+    }
+
     .jkl-tag {
         margin: 1px;
```

Med endringen på inputfeltet over kan jeg ikke klikke på tags for å fjerne dem lenger. Ser ut som det løser seg med en `z-index: 1` her.

```suggestion
        margin: 1px;
        z-index: 1;
```